app/controllers/admin/home_controller.rb

```diff
@@ -35,8 +35,8 @@ def stat_locals
     def stat
       @stat ||= {
         events_by_status:   Event.group(:status).count,
-        users_by_months:    User.group_by_month(:created_at).count,
-        talks_by_months:    Talk.group_by_month(:created_at).count,
+        users_by_months:    User::PerMonth.call,
```

Comment:
First question when reading `PerMonth` - per month what?

I think better name for this will be `IncreasePerMonth` (same for talks)